scripts/serve-pimcore5.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# install apache and bzip PHP extension
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
I see your point - maybe make the apache installation an extra script which can be required from types building on apache?